library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1320,6 +1320,54 @@ def SetServicesForZones(services_ids, firewall_zones, new_status)
       nil
     end
 
+    # Function returns actual state of Masquerading support.
+    # In FirewallD, masquerade is enabled per-zone so this
+    # function has to iterate through all the zones and return
+    # a collective boolean value. This API is probably not very
```

Comment:
In the UI, "Masquerade" is a single checkbox.
The compatibility code here choses ANY for the getter and ALL for the setter. Wouldn't it be better to just forward to a particular zone, say "internal" ?